eos-updater-flatpak-installer/docs/eos-updater-flatpak-installer.8

```diff
@@ -0,0 +1,100 @@
+.\" Manpage for eos-updater-flatpak-installer.
+.\" Documentation is under the same licence as the eos-updater package.
+.TH man 8 "13 Apr 2017" "1.0" "eos\-updater\-flatpak\-installer man page"
+.\"
+.SH NAME
+.IX Header "NAME"
+eos\-updater\-flatpak\-installer — Endless OS Updater - Flatpak Installer
+.\"
+.SH SYNOPSIS
+.IX Header "SYNOPSIS"
+.\"
+\fBeos\-updater\-flatpak\-installer [\-m \-\-mode\fB] [\-p \-\-pull\fB]
+.\"
+.SH DESCRIPTION
+.IX Header "DESCRIPTION"
+.\"
+\fBeos\-updater\-flatpak\-installer\fP installs downloaded flatpaks on Endless OS
+updates upon rebooting into the new update. It is the part of the system that
+ensures that when new flatpaks are installed on OS Updates, they are
+installed "seamlessly", only appearing when rebooting into the new system.
+.PP
+.SH OPTIONS
+.IX Header "OPTIONS"
+.\"
+.IP "\fB\-m\fP, \fB\-\-mode\fP"
+Which mode to run the flatpak installer in. When set to \fBperform\fP, the
+installer will examine the list of applications to install in
+$(datadir)/eos-application-tools/flatpak-autoinstall.d/*,
```

Comment:
The reader doesn’t know what `$(datadir)` will be, so just hard-code it. (We could technically substitute it into the man page as part of the build process, but that’s too much faff to bother with.)

Reply:
Also, don’t bother with the `*`: just ending the path with `/` signifies it’s a directory and it should be obvious you mean all the files in that directory.

Reply:
Also, please format paths as `\fI/path/blah/blah/\fP` to make them stand out a bit from the surrounding prose.